CHECKERS.md

```diff
@@ -0,0 +1 @@
+../cve_bin_tool/checkers/README.md
```

Comment:
I don't think we want this file as a top level file when there's already a link for it within `doc/`